packages/lib/slugify.ts

```diff
@@ -5,10 +5,9 @@ export const slugify = (str: string) => {
     .normalize("NFD") // Normalize to decomposed form for handling accents
     .replace(/\p{Diacritic}/gu, "") // Remove any diacritics (accents) from characters
     .replace(/[^\p{L}\p{N}\p{Zs}\p{Emoji}]+/gu, "-") // Replace any non-alphanumeric characters (including Unicode) with a dash
```

Comment:
I verified this regex, it should not have ideally allowed # character. I am not sure why it is doing so. Therefore had to add an explicit check to replace # character